src/ckeditor.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* global CKEDITOR */
+
+import getEditorNamespace from './utils/get-editor-namespace.js';
+
+export default {
+	name: 'ckeditor',
+
+	render( createElement ) {
+		return createElement( 'div', {}, [
+			createElement( this.tagName )
```

Comment:
Is there a reason for wrapping it with empty `div`? Not sure but you could probably just use:
`return createElement( this.tagName)`
and replace `$el.firstElementChild` with `$el`.

Reply:
We need two elements here. One which is component wrapper, so all component elements are grouped within single element and another element which to be passed to `CKEDITOR.replace`, let's call it `textarea` although it can be anything else.

When returning only `textarea` then editor is appended to parent component not the editor component. This breaks the whole purpose of using component based frameworks. Additionally it might lead to issues for example `v-show="false"` directive adds `display:none` to components outer element, so it probably won't hide editor which rendered outside of the component.

Reply:
I see, thanks for clarifying that.